package.json

```diff
@@ -1,8 +1,20 @@
 {
-  "name": "loggly-winston-bulk",
-  "version": "1.0.1",
+  "name": "winston-loggly-bulk",
+  "version": "1.3.4",
   "description": "A Loggly transport for winston",
-  "author": "Shweta Jain <sjain@loggly.com>",
+  "author": "Loggly <sjain@loggly.com>",
+  "contributors": [
+    {
+      "name": "Loggly",
+      "email": "sjain@loggly.com",
+      "url": "https://github.com/loggly"
+    },
+    {
```

Comment:
its good practice to add Charlie Robbins to this list, as he is the main contributer